winrm/client.go

```diff
@@ -18,10 +19,22 @@ type Client struct {
 	password  string
 	useHTTPS  bool
 	url       string
+	err       error
 	http      HttpPost
 	transport http.RoundTripper
 }
 
+// Map of known terminating errors, this will be utilized for returning error codes and error messages on
+// fatal errors
+var errorCode = map[string]int{
```

Comment:
I think instead of using that much memory just to make map of errors , will be better to make a separated file with declared errors like:

```go
var (
  errEOF = errors.New("A connection terminated unexpectedly, error while sending request to endpoint")
  errorTimeout = errors.New("Operation timeout because there was no command output")
)
```
Go has these types of error declarations inside stdlib for example witch in my opinion is much more nicer than using a hole map.